samples/golang/mywebapi/src/app/hello.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
```

Comment:
The indentation of 6 characters seems non-standard. I'd choose 4.

Reply:
I see indentation of a tab (4 spaces) only.

Reply:
Ah, tabs. Ok, Go does expect to use tabs.